frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/request.ts

```diff
@@ -22,7 +21,7 @@ const getDeviceTypes = (deviceType: string[]) => {
 };
 
 export const getLocalVolumeSetRequestData = (
-  state: State,
+  state: any,
```

Comment:
I would need to refactor at more places, hence passing "any" for now.
Noting as tech debt

Reply:
Please add a comment in the code as a todo.